src/Bloc-Examples/BlBackgroundTest.class.st

```diff
@@ -1,30 +1,13 @@
-"
-# Backgrounds
-
-The class {{gtClass:BlBackground}} defines the background of a Bloc element. As this is an abstract class, the actual concrete types of backgrounds are defined by its subclasses. 
-
-There are four main types of backgrounds:
-- {{gtClass:BlPaintBackground}}: color background defined using a {{gtClass:BlPaint}} object;
-- {{gtClass:BlImageBackground}}: image background defined using a {{gtClass:Form}} object;
-- {{gtClass:BlTransparentBackground}}: a special case of a transparent background;
-- {{gtClass:BlCompositeBackground}}: a background that provided the ability to compose multiple other {{gtClass:BlBackground}} objects.
-
-## Paint Backgrounds
-The simplest of the available backgrounds is the paint background. A paint can be either a simple color or a type of gradient.
```

Comment:
I guess no problem to loose this comment, as it's explained in BlBackground already,